<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqSQVolumeSourceForm</class>
 <widget class="QWidget" name="pqSQVolumeSourceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QCheckBox" name="immediateMode">
       <property name="text">
        <string>Immediate Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>O</string>
       </property>
       <property name="buddy">
        <cstring>o_x</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="o_x"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="o_y"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="o_z"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>P_1</string>
       </property>
       <property name="buddy">
        <cstring>p1_x</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="p1_x"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="p1_y"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="p1_z"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>P_2</string>
       </property>
       <property name="buddy">
        <cstring>p2_x</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="p2_x"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="p2_y"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="p2_z"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>P_3</string>
       </property>
       <property name="buddy">
        <cstring>p3_x</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="p3_x"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="p3_y"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="p3_z"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="dx">
       <property name="text">
        <string>1</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="dy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="dz">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Resolution</string>
       </property>
       <property name="buddy">
        <cstring>res_x</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="res_x">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="res_y">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="res_z">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Spacing</string>
       </property>
       <property name="buddy">
        <cstring>dx</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="aspectLock">
         <property name="text">
          <string>lock aspect ratio</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="spacingLock">
         <property name="text">
          <string>lock grid spacing</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Coord-sys</string>
       </property>
       <property name="buddy">
        <cstring>coordStatus</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="coordStatus">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>NCells</string>
       </property>
       <property name="buddy">
        <cstring>nCells</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nCells">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Dimensions</string>
       </property>
       <property name="buddy">
        <cstring>dim_x</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dim_x">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="dim_y">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="dim_z">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="restore">
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>136</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>o_x</tabstop>
  <tabstop>o_y</tabstop>
  <tabstop>o_z</tabstop>
  <tabstop>p1_x</tabstop>
  <tabstop>p1_y</tabstop>
  <tabstop>p1_z</tabstop>
  <tabstop>p2_x</tabstop>
  <tabstop>p2_y</tabstop>
  <tabstop>p2_z</tabstop>
  <tabstop>p3_x</tabstop>
  <tabstop>p3_y</tabstop>
  <tabstop>p3_z</tabstop>
  <tabstop>dx</tabstop>
  <tabstop>dy</tabstop>
  <tabstop>dz</tabstop>
  <tabstop>res_x</tabstop>
  <tabstop>res_y</tabstop>
  <tabstop>res_z</tabstop>
  <tabstop>aspectLock</tabstop>
  <tabstop>spacingLock</tabstop>
  <tabstop>coordStatus</tabstop>
  <tabstop>nCells</tabstop>
  <tabstop>dim_x</tabstop>
  <tabstop>dim_y</tabstop>
  <tabstop>dim_z</tabstop>
  <tabstop>restore</tabstop>
  <tabstop>save</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>aspectLock</sender>
   <signal>toggled(bool)</signal>
   <receiver>res_y</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aspectLock</sender>
   <signal>toggled(bool)</signal>
   <receiver>dy</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aspectLock</sender>
   <signal>toggled(bool)</signal>
   <receiver>res_z</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aspectLock</sender>
   <signal>toggled(bool)</signal>
   <receiver>dz</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
